application-templates/starter-typescript/package.json

```diff
@@ -39,23 +39,23 @@
     "@commercetools-test-data/channel": "4.11.1",
     "@commercetools-test-data/commons": "4.11.1",
     "@commercetools-test-data/core": "4.11.1",
-    "@commercetools-uikit/constraints": "^16.0.0",
-    "@commercetools-uikit/data-table": "^16.0.0",
-    "@commercetools-uikit/flat-button": "^16.0.0",
-    "@commercetools-uikit/grid": "^16.0.0",
-    "@commercetools-uikit/hooks": "^16.0.0",
-    "@commercetools-uikit/icons": "^16.0.0",
-    "@commercetools-uikit/link": "^16.0.0",
-    "@commercetools-uikit/loading-spinner": "^16.0.0",
-    "@commercetools-uikit/localized-text-field": "^16.0.0",
-    "@commercetools-uikit/localized-text-input": "^16.0.0",
-    "@commercetools-uikit/notifications": "^16.0.0",
-    "@commercetools-uikit/pagination": "^16.0.0",
-    "@commercetools-uikit/select-field": "^16.0.0",
-    "@commercetools-uikit/spacings": "^16.0.0",
-    "@commercetools-uikit/text": "^16.0.0",
-    "@commercetools-uikit/text-field": "^16.0.0",
-    "@commercetools-uikit/text-input": "^16.0.0",
+    "@commercetools-uikit/constraints": "^16.4.0",
```

Comment:
Perhaps this is not needed as we're using the _caret_ in the versions definition so running `pnpm install` again will automatically update to the latest `16.x` version.

Reply:
Yes that's true, this is relevant for consumers though. Still, having an update here is fine as sometimes dependent package versions are not updated automatically.